File: HealthEase/HealthEase.Services/Database/Patient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthEase.Services.Database
{
    public partial class Patient : BaseEntity
    {
        [Key]
        public int PatientId { get; set; }
        [MaxLength(100)]
        public string Username { get; set; } = null!;
        [MaxLength(100)]
        public string FirstName { get; set; } = null!;
        [MaxLength(100)]
        public string LastName { get; set; } = null!;
        [EmailAddress]
        public string Email { get; set; } = null!;
        public string? PhoneNumber { get; set; }
        public string PasswordHash { get; set; } = null!;
        public string PasswordSalt { get; set; } = null!;
        public byte[]? ProfilePicture { get; set; }
        public DateTime RegistrationDate { get; set; }
        public bool isActive { get; set; }
        public virtual ICollection<Appointment> Appointments { get; set; } = new List<Appointment>();
        public virtual ICollection<Review> Reviews { get; set; } = new List<Review>();
    }
}

File: HealthEase/UI/healthease_desktop/lib/models/patient.dart

import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';
part 'patient.g.dart';

@JsonSerializable()
class Patient {
  int? patientId;
  String? firstName;
  String? lastName;
  String? email;
  String? phoneNumber;
  String? username;
  String? profilePicture;
  String? registrationDate;
  bool? isActive;

  Patient();

  factory Patient.fromJson(Map<String, dynamic> json) =>
      _$PatientFromJson(json);

      Map<String, dynamic> toJson() => _$PatientToJson(this);
}